R/janitor.R

```diff
@@ -1,24 +1,7 @@
-#' janitor
-#'
-#' janitor has simple little tools for examining and cleaning dirty data.
-#'
-#' @section Main functions:
-#' The main janitor functions can: perfectly format data.frame
-#' column names; provide quick counts of variable combinations (i.e.,
-#' frequency tables and crosstabs); and explore duplicate records. Other
-#' janitor functions nicely format the tabulation results. These
-#' tabulate-and-report functions approximate popular features of SPSS and
-#' Microsoft Excel.
-#'
 #' @section Package context:
-#' This package follows the principles of the "tidyverse" and works
-#' well with the pipe function `\%>\%`.
-#'
-#' janitor was built with beginning-to-intermediate R users in mind
-#' and is optimized for user-friendliness.  Advanced users can do most
-#' things covered here, but they can do it faster with janitor and save
-#' their thinking for more fun tasks.
-#'
+#' Advanced users can do most things covered here, but they can do it
+#' faster with janitor and save their thinking for more fun tasks.
+#' @keywords internal
 "_PACKAGE"
```

Comment:
Note that the difference in .Rd is minimal, because by default `"_PACKAGE"` uses the package description from `DESCRIPTION`

Reply:
Nice, thanks for explaining how this works.